src/Vegetation/Roots.jl

```diff
@@ -357,18 +366,18 @@ function flow_out_roots(
     p::ClimaCore.Fields.FieldVector,
     t::FT,
 )::FT where {FT}
-    @unpack a_root, b_root, K_max_root_moles, size_reservoir_stem_moles =
+    @unpack a_variables, b_variables, K_max_moles, size_reservoir_moles =
         model.parameters
-    p_stem = theta_to_p(Y.vegetation.rwc[1] / size_reservoir_stem_moles)
+    p_stem = theta_to_p(Y.vegetation.rwc[1] / size_reservoir_moles[:stem])
```

Comment:
I dont think we can hardcode this. If n_leaf = 1, n_stem = 0, then the size should be for the leaf.

Reply:
a solution to this could be adding an argument to the flow_out_roots function: the type of the first compartment above the soil?